qa/suites/crimson-rados/seastore/basic/tasks/rados_api_tests.yaml

```diff
@@ -0,0 +1,28 @@
+overrides:
+  ceph:
+    log-ignorelist:
+    - reached quota
+    - but it is still running
+    - overall HEALTH_
+    - \(POOL_FULL\)
+    - \(SMALLER_PGP_NUM\)
+    - \(CACHE_POOL_NO_HIT_SET\)
+    - \(CACHE_POOL_NEAR_FULL\)
+    - \(POOL_APP_NOT_ENABLED\)
+    - \(PG_AVAILABILITY\)
+    - \(PG_DEGRADED\)
+    conf:
+      client:
+        debug ms: 1
+      mon:
+        mon warn on pool no app: false
+      osd:
+        osd class load list: "*"
+        osd class default list: "*"
+        osd blocked scrub grace period: 3600
+tasks:
+- workunit:
+    clients:
+      client.0:
+        - rados/test.sh
+          - rados/test_pool_quota.sh
```

Comment:
Wrong indentation.
Did you try running against it?

Reply:
yes, every test.sh test case passed teuthology test,  But I can't run them at the same time, I meet pg peering issue, it is not the api test itself bug, it is crimson bugs.